### PR TITLE
fix: quickstart change and profile selection

### PR DESCRIPTION
**Pull request by nikhname**
*Issue #, if available:*

*Description of changes:*
Remove auth from quickstart and change api name to amplifyDatastore
Ask question for profile if default doesn't exist
By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: packages/amplify-app/src/scripts/amplify-push.js [hunk 1]
```diff
@@ -24,14 +25,25 @@ function getNamedProfiles() {
   return namedProfiles;
 }
 
-function checkIfProfileExists(profileToUse) {
+async function checkIfProfileExists(profileToUse) {
```
Comment (yuth) [inline packages/amplify-app/src/scripts/amplify-push.js]: functin name should be changed. Better yet split this into 2 functions. One for checking and other for asking the user which profile they want to use if selected profile is missing in the computer

File: packages/amplify-app/src/scripts/amplify-push.js [hunk 2]
```diff
@@ -24,14 +25,25 @@ function getNamedProfiles() {
...
-
-  if (!namedProfiles || !namedProfiles[profileToUse]) {
-    console.log(`Profile with name ${profileToUse} not found. Please create one`);
-    return false;
+  if (Object.keys(namedProfiles).length) {
+    if (namedProfiles[profileToUse]) {
+      return profileToUse;
+    } else if (Object.keys(namedProfiles).length === 1) {
+      return Object.keys(namedProfiles)[0];
+    }
```
Comment (yuth) [inline packages/amplify-app/src/scripts/amplify-push.js]: Do we want to still ask the customer if they want to use the only profile available?

Comment (nikhname) [reply packages/amplify-app/src/scripts/amplify-push.js]: They are implicitly choosing that profile if they are going through this flow and that is the only profile that exists

Comment (nikhname) [reply packages/amplify-app/src/scripts/amplify-push.js]: Changing to ask even if is just one profile

File: packages/amplify-app/src/scripts/amplify-push.js (hunk 1 above)
Comment (nikhname) [reply packages/amplify-app/src/scripts/amplify-push.js]: Separating into two functions

File: packages/amplify-app/src/xcodeHelpers.js [hunk 3]
```diff
@@ -5,7 +5,7 @@ const fs = require('fs-extra');
 async function getXcodeProjectDir() {
   const EXTENSION = '.xcodeproj';
   const files = fs.readdirSync(process.cwd());
-  const targetFiles = files.filter(function(file) {
+  const targetFiles = files.filter(function extenstionFilter(file) {
```
Comment (yuth) [inline packages/amplify-app/src/xcodeHelpers.js]: use arrow functions for these things when possible, so this does not get rebound

**Pull request merged**